app/services/update_interview.rb

```diff
@@ -23,12 +23,14 @@ def save!
       course_option_id: interview.application_choice.offered_option.id,
     )
 
-    interview.update!(
-      provider: provider,
-      date_and_time: date_and_time,
-      location: location,
-      additional_details: additional_details,
-    )
+    interview.provider = provider
+    interview.date_and_time = date_and_time
+    interview.location = location
+    interview.additional_details = additional_details
+
+    return unless interview.changed?
+
+    interview.save!
```

Comment:
an other way of achieving this is still running the `update!` as it will only be executed if there are changes in the fields, and  returning unless `interview.saved_change_to_updated_at?`.

Reply:
I'll go with what we've got now – I find it a bit more readable